app/download/views.py

```diff
@@ -110,3 +115,44 @@ def get_document_metadata(service_id, document_id):
     response.headers['X-Robots-Tag'] = 'noindex, nofollow'
     response.headers['Referrer-Policy'] = 'no-referrer'
     return response
+
+
+@download_blueprint.route('/services/<uuid:service_id>/documents/<uuid:document_id>/authenticate', methods=['POST'])
+def authenticate_document(service_id, document_id):
```

Comment:
`authenticate_document` sounds like we were checking the document. Maybe `authenticate_recipient` or `authenticate_document_access` would be more accurate?

Reply:
Good call - have renamed to `authenticate_access_to_document`